csharp/src/AddressBook/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("2.4.1.555")]
 
-[assembly: AssemblyVersion("2.4.1.555")]
-[assembly: AssemblyFileVersion("2.4.1.555")]
\ No newline at end of file
+[assembly: AssemblyVersion("3.0.0.1")]

```

Comment:
Hm, something that need to be discussed, this is a bit tricky.

AFAIK, the release is going to be 3.0.0-alpha4 or 3.0.0-beta. now we don't have a good way of representing that with the assembly version. There's two options I can think of and none of them is perfect:
1. just set to 3.0.0.0 and leave it there until protobuf project as a whole becomes 3.x.y and then change it to reflect that. The downside is multiple releases of C# would have the same assembly version, which I don't think is acceptable.

2. set to 3.0.0.x  so that the "3.0.0" always matches the version of protobuf release (we should stay consistent). Then we have the build number left to express version of C#. Given we need to represent things like alpha-3 and alpha-4 (that are numerically lower than beta and GA version), we just start with some number (here 1) and increment it with every release regardless of what the release label is. That way, once it's GA (and protobuf project will be at straight 3.0.0), we might have a assembly version like 3.0.0.5 or so.